modules/fedimint-wallet-tests/tests/tests.rs

```diff
@@ -977,11 +991,15 @@ mod fedimint_migration_tests {
                     WalletCommonInit::KIND.to_string(),
```

Comment:
The `apply_migrations` 2 lines above here, should be different `apply_migrations`, one that would hardcode the module id for the purpose of testing, while the "real" `apply_migrations` will take it as an argument (from the global code).

Reply:
Are you suggesting to change hardcode the module instance id here? You're right that we don't need to use the `LEGACY` constants, the number just needs to be consistent across the snapshot test and the validate test

Reply:
Module developer writing migration tests should not need to specify any module ID or even be aware there's such a thing, yes. @m1sterc001guy

Reply:
Removed all instances of module ids from the module db migration tests